src/includes/getting-started-config/javascript.react.mdx

```diff
@@ -0,0 +1,121 @@
+You should initialize the Sentry React SDK as soon as possible during your application load up, before initializing React:
+
+```JSX
+import React from "react";
+import ReactDOM from "react-dom";
+import * as Sentry from "@sentry/react";
+import { Integrations } from "@sentry/tracing";
+import App from "./App";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___" ,
+  integrations: [new Integrations.BrowserTracing()],
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0,
+});
+
+ReactDOM.render(<App />, document.getElementById("root"));
+
+// Can also use with React Concurrent Mode
+// ReactDOM.createRoot(document.getElementById('root')).render(<App />);
+```
+
+Once this is done, all unhandled exceptions are automatically captured by Sentry.
+
+### Add React Error Boundary
+
+If you're using React 16 or above, Error Boundaries are an essential tool for defining your application's behavior in the face of errors. The `@sentry/react` package exposes an error boundary component that automatically sends JavaScript errors from inside a React component tree to Sentry. You can use the error boundary just like a regular React component.
+
+```JSX
+import React from "react";
+import * as Sentry from "@sentry/react";
+
+<Sentry.ErrorBoundary fallback={"An error has occurred"}>
+  <Example />
+</Sentry.ErrorBoundary>;
+```
```

Comment:
Suggest moving this to a subpage and keeping only the statement "If you're using React 16 or above, Error Boundaries are an essential tool for defining your application's behavior in the face of errors" and creating a link to that page with the [Error Boundaries] text.